src/components/MetadataCollections/MetadataCollections.js

```diff
@@ -238,12 +290,13 @@ class MetadataCollections extends React.Component {
                             inputRef={this.searchField}
                             name="query"
                             onChange={getSearchHandlers().query}
-                            onClear={getSearchHandlers().reset}
+                            // onClear={getSearchHandlers().reset}
+                            onClear={() => this.handleClearSearch(getSearchHandlers(), onSubmitSearch(), searchValue)}
                             value={searchValue.query}
                           />
                           <Button
                             buttonStyle="primary"
-                            disabled={!searchValue.query || searchValue.query === ''}
+                            // disabled={!searchValue.query || searchValue.query === ''}
```

Comment:
I think it is good practice, if buttons are disabled, if their use has no impact (e.g. the form did not change).

Reply:
Will be fixed in different ticket.